### ISSUE TITLE
portrait 2: Correct set up for registration marks (Solved)

### ISSUE THREAD

**Issue by bmsleight** [fablabnbg/inkscape-silhouette#128]
TLDR; Load media, switch-off, switch-on load media. Use default setting. 

inkscape-silhouette is excellent. 

HI All,

I have portrait 2 now working with registration marks.

Using the standard 
    X mark distance  190)
    Y mark distance  277)
    Position of regmark from document left: 10
    Position of regmark from document top: 10

I tried using the offsets and/or not using a cutting map.  After a lot of trial and error, I need to have the media loaded further into the machine than the default load for the registration marks to work. Set as Auto-registration and the correct distances. (A little counter initiative)

Solution to get a "double load": Load media, switch-off, switch-on, load media. Use default setting.

Not sure if it is just my machine.

**Comment by cappy0815** [comment]
Hi Brendan,
thanks for you post.
I am just about to burn my portrait2 ... :-(

The registration marks will not be found.
I'm using the same settings (190,277,10,10) with an offset -10,-10

On starting the plot ... the head is moving to the left and running into an error while the head is bumping the housing until switching the machine off.

Unfortunately I do not understand your double load procedure!?

[edit a few min later]
got it ... the "double load" is working well. Now the machine is able to finde the marks.
It is important to set the offset to 0 !!!
[/edit]
